pkg/cmd/server/server.go

```diff
@@ -604,12 +603,13 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 	backupControllerRunInfo := func() controllerRunInfo {
 		backupper, err := backup.NewKubernetesBackupper(
 			s.veleroClient.VeleroV1(),
-			s.discoveryHelper,
-			client.NewDynamicFactory(s.dynamicClient),
-			podexec.NewPodCommandExecutor(s.kubeClientConfig, s.kubeClient.CoreV1().RESTClient()),
-			s.resticManager,
-			s.config.podVolumeOperationTimeout,
-			s.config.defaultVolumesToRestic,
+			s.mgr.GetClient(),
+			// Because we don't want to run any hook commands.
```

Comment:
fine for, but can we reduce the diff by just handing over the two clients and they are unused in our case? (then we only have to document it were we commented out the usage of this stuff)